runtime/help/commands.md

```diff
@@ -31,9 +31,15 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `quit`: quits micro.
 
-* `goto 'line'`: jumps to the given line number. A negative number can be
-   passed to jump inward from the end of the file; for example, -5 jumps
-   to the 5th-last line in the file.
+* `goto '[-+]?line'(:'[-+]?col')?`: Jumps to the given line (and column) number.
+   Both `line` and `col` are integers and can be prefixed with `+` or `-`.
+   The latter will be used to perform relative jumps from the current position
+   of the cursor.
+
+   Note:
+   The previous provided feature to jump inward from the end of the file needs a
+   new prefix `~` in favor to the relative jumps.
```

Comment:
I don't think the documentation should refer to the history of changes, it will confuse the readers who don't know the context. Just document `~` together with `+` and `-`.